tests/behaviour/cube.feature

```diff
@@ -251,10 +380,179 @@ Feature: Cube!
-      ]
+    [
+      "Dimension",
+      "Value",
+      "Measure",
+      "Unit"
+    ]
+    """
+    Then The cube data should match
+    """
+    [
+      {
+        "Dimension":"a",
+        "Value":0,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"b",
+        "Value":1,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"c",
+        "Value":2,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"d",
+        "Value":3,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"e",
+        "Value":4,
+        "Measure":"length",
+        "Unit":"cm"
+      }
+    ]
+    """
+
+    Scenario: Output a cube when the code list is defined using code list config schema v1.0 and when there are references to concepts defined elsewhere.
+    Given The config json file "v1.1/cube_config_with_concepts_defined_elsewhere.json" and the existing tidy data csv file "v1.1/data.csv"
+    When The cube is created
+    Then The cube Metadata should match
+      """
+      {
+        "title": "Test cube config",
+        "identifier": None,
+        "summary": "Test cube config summary",
+        "description": "Test cube config description",
+        "creator_uri": "http://statistics.data.gov.uk",
+        "publisher_uri": "http://statistics.data.gov.uk",
+        "landing_page_uris": [],
+        "theme_uris": [],
+        "keywords": ["Test cube"],
+        "dataset_issued": "2022-04-08",
+        "dataset_modified": None,
+        "license_uri": "https://creativecommons.org/licenses/by/4.0/",
+        "public_contact_point_uri": None,
+        "uri_safe_identifier_override": None
+      }
+      """
+    Then The cube Columns should match
+    """
+    [
+      "Dimension",
+      "Value",
+      "Measure",
+      "Unit"
+    ]
+    """
+    Then The cube data should match
+    """
+    [
+      {
+        "Dimension":"a",
+        "Value":0,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"b",
+        "Value":1,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"c",
+        "Value":2,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"d",
+        "Value":3,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"e",
+        "Value":4,
+        "Measure":"length",
+        "Unit":"cm"
+      }
+    ]
+    """
+    
+    Scenario: Output a cube when an inline code list is defined using code list config schema v1.0 and when there are references to concepts defined elsewhere.
```

Comment:
This one should really be asserting that the code list CSV file which has been written to disk has the same as column containing the expected values.

Reply:
@robons The cols and their content are tested in the following unit tests at test_code_list_config.py: test_code_list_config_model, test_code_list_config_model_without_schema, test_code_list_config_model_with_hierarchy.